### PR TITLE
Improve linux compatibility. Fixes #135

### LINKED ISSUE #135: [coc.vim] javascript file not found, please compile the code or use release branch
[Solution by kenden](https://github.com/neoclide/coc.nvim/issues/651#issuecomment-549770055)

> It's not an issue with Vundle.
> I removed it, and installed vim-plug, and got the same issue.
> With Vundle or vim-plug, this command solved it (edit: from comment above)
> 
> ```
> :call coc#util#install()
> ```
> 
> Using a mac

Maybe it has to be added to the readme.

### PR DESCRIPTION
**Pull request by ajr-dev**
modify brewfile to work in linux
remove duplicate code in install.sh
add branch release to coc fixes javascript file not found
remove delay in escape sequences

I didn't see I had changed your name in `git/gitconfig.symlink`, so I had to repeat the changes from #134.

### CODE REVIEW

**Comment by nicknisi** [comment]
I did not know about conditionals in Brewfile!

File: config/nvim/init.vim [hunk 1]
```diff
@@ -60,7 +68,11 @@ call plug#begin('~/.config/nvim/plugged')
     set showbreak=↪
     set autoindent " automatically set indent of new line
     set ttyfast " faster redrawing
-    set diffopt+=vertical,iwhite,internal,algorithm:patience,hiddenoff
+    if g:os == "Darwin"
+        set diffopt+=vertical,iwhite,internal,algorithm:patience,hiddenoff
+    elseif g:os == "Linux"
+        set diffopt=vertical,iwhite
```
Comment (nicknisi) [inline config/nvim/init.vim:74]: Why would this value be based on OS?

Comment (ajr-dev) [reply config/nvim/init.vim:74]: What I would like to have done is add the options available in the nvim installation but I couldn't find the way, so I added the options available in the package manager installation of nvim in ubuntu testing for the os. Definitely not the best solution but I couldn't find a better one.

**Comment by nicknisi** [comment]
Closed in favor of #147. Thank you!

**Pull request closed without merging**